### PR TITLE
Display the 3 most recent blog posts on a profile.

### PR DESCRIPTION
**Pull request by st23am**

### CODE REVIEW

File: engines/blog_app/app/models/blog_app/post.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def self.tags
     end
 
     def self.published
-      where('published_at <= ?', Time.now).order('published_at DESC')
+      where('published_at <= ?', Time.now).order('published_at ASC')
```
Comment (cdmwebs) [inline engines/blog_app/app/models/blog_app/post.rb]: You don't have to write ASC. It's the default.

**Comment by cdmwebs** [comment]
:+1:

**Comment by st23am** [comment]
Why not pull request?

**Comment by cdmwebs** [comment]
Merged in 760ee0.

**Pull request closed without merging**